From b16833392b74826f8cd437675fc29e4a0b88efcd Mon Sep 17 00:00:00 2001
From: Randy Barlow <randy@electronsweatshop.com>
Date: Sun, 4 Sep 2022 00:03:33 -0400
Subject: [PATCH] Only build ssh keys when needed

I believe the test/remote folder only needs to be setup for
test/test_remote.sh. Prior to this commit, it was being built during
make, rather than just during make check. This commit adjusts things so
that the test/remote folder is only generated during make check, and
only when test/test_remote.sh is being executed.

Fixes #1040

Signed-off-by: Randy Barlow <randy@electronsweatshop.com>
---
 test/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/Makefile.am b/test/Makefile.am
index fd09a656..457f3cb0 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -478,7 +478,7 @@ DISTCLEANFILES = \
 	empty \
 	scripts-empty
 
-all-local: remote/ssh_host_dsa_key remote/ssh_host_rsa_key remote/id_rsa
+test_remote.sh.log: remote/ssh_host_dsa_key remote/ssh_host_rsa_key remote/id_rsa
 
 distclean-local:
 	$(RM_V)rm -rf remote remote-tmp not:a:remote:dir
-- 
2.37.2

